src/main/java/GiaiThuaBai8.java

```diff
@@ -0,0 +1,13 @@
+public class GiaiThuaBai8 {
+
+    public int layGiaiThuaCuaSo(int number) {
+        if(number <= 0) {
+            return 0;
+        }
+        int p = 1;
+        for(int i = number; i>= 2; --i) {
```

Comment:
Nên viết vòng for tăng thay vì for giảm (tăng tính dễ đọc)

Reply:
done.